jetbrains-core/src/software/aws/toolkits/jetbrains/core/filtering/FilterManager.kt

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.filtering
+
+import com.intellij.openapi.project.Project
+
+interface FilterManager {
```

Comment:
This is confusing how it differs from Filter.

The lines between this, Filter, FilterManagerService are a bit too blurred from the API definitions.

This seems to duplicate all the fields, so its not clear why Filter exists